src/Functions/LowerUpperUTF8Impl.h

```diff
@@ -2,6 +2,7 @@
 #include <Columns/ColumnString.h>
 #include <Functions/LowerUpperImpl.h>
 #include <base/defines.h>
+#include <base/find_symbols.h>
```

Comment:
Many developers do "slim" builds, e.g.

```bash
cmake -DENABLE_EMBEDDED_COMPILER=0 -DENABLE_LIBRARIES=0 -DENABLE_LIBURING=1 -DCMAKE_BUILD_TYPE=Debug -S . -B build
```

This build now broke as ifdefs for `USE_ICU` are missing  :-) Like to fix it?